staging/src/k8s.io/api/flowcontrol/v1beta3/register.go

```diff
@@ -54,5 +55,11 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&PriorityLevelConfigurationList{},
 	)
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
+
+	// Registers the lifecycle of the resource types, which is checked to make sure a gvr is not available before its type is introduced or after it is removed.
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("flowschema"), &FlowSchema{})
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("flowschemas"), &FlowSchemaList{})
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("prioritylevelconfiguration"), &PriorityLevelConfiguration{})
+	scheme.SetResourceLifecycle(SchemeGroupVersion.WithResource("prioritylevelconfigurations"), &PriorityLevelConfigurationList{})
```

Comment:
Can this be auto-generated?

Reply:
Maybe. But we don't have time to add any machinery in this release. We will add more machinery regarding comp version in the following releases.